.github/workflows/publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to PyPI
+on:
+  push:
```

Comment:
Does this run only when main is updated, or on any push?

Reply:
I am testing to see how this works right now. I wouldn't review this quite yet

Reply:
Aye aye cap'n